Netcracker/KubeMarine
# [CPDEV-93148] [CPDEV-93149] Clarify kubernetes audit rules

### Description
* There are not logged, installed by default resources.
* There are some enabled audit logs that hardly have any value but produce constant garbage.

Fixes #511
Fixes #512

### Solution
* Enable audit logging of Calico resources.
* Disable logging of changes in `ingress-controller-leader` ConfigMap.
* Do not log checking access by kubelet.

### How to apply
Run `kubemarine migrate_kubemarine --force-apply enable_calico_audit`.

### Test Cases

**TestCase 1**

Check audit logs of Calico resources.

Steps:

1. Install cluster.
2. `grep projectcalico /var/log/kubernetes/audit/audit.log`

Results:

| Before | After |
| ------ | ------ |
| No events | Events are present |

**TestCase 2**

Change Calico resources using `kubectl` and check audit logs.

Steps:

1. Install cluster with `calico.apiserver.enabled=true`.
2. Get `kubectl get ippools.projectcalico.org default-ipv4-ippool -o yaml`, change, and apply using `kubectl`.
3. Find audit event for `projectcalico.org` group in  `/var/log/kubernetes/audit/audit.log`

ER: Event is present.

**TestCase 3**

Check update logs of nginx-ingress leader ConfigMap.

Steps:

1. Install cluster of Kubernetes versions v1.24.11, v1.26.3, v1.26.4, v1.28.0.
2. Run `tail -f /var/log/kubernetes/audit/audit.log | grep ingress-controller-leader` or `tail -f /var/log/kubernetes/audit/audit.log | grep ingress-nginx-leader` on each control plane node.

Results:

| Before | After |
| ------ | ------ |
| On v1.24.11 update events constantly appear | No events |

**TestCase 4**

Migrate Kubemarine.

Steps:

1. Install cluster using **old** Kubemarine.
2. Run `kubemarine migrate_kubemarine --force-apply enable_calico_audit` using new Kubemarine.

ER: The policy is updated.

### Checklist
